### PR TITLE
Add error handling for login

### PR DESCRIPTION
**Pull request by samann**

### CODE REVIEW

**Comment by aclissold** [comment]
"If not check for errors in input" doesn't make sense

File: Lunch/LoginViewController.swift [hunk 1]
```diff
@@ -53,4 +55,23 @@ class LoginViewController: UIViewController {
             lunchTableViewController.loadObjects()
         }
     }
+
+    func checkForErrorsInInput(username: String, password: String) -> Bool {
+        var hasErrors = false
+        var alertView = UIAlertController(title: "Error", message: "", preferredStyle: .Alert)
+        alertView.addAction(UIAlertAction(title: "Ok", style: .Cancel, handler: nil))
+        let name = username, psswrd = password
```
Comment (aclissold) [inline Lunch/LoginViewController.swift]: Why?

Comment (samann) [inline Lunch/LoginViewController.swift]: I was doing something different before. I'll remove it.

**Comment by aclissold** [comment]
I think "Address comments" is a bad commit name; the version control system shouldn't know about the issue system and the git history should always have meaningful but succinct commit messages.

**Pull request merged**